shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -803,6 +803,7 @@ - (instancetype)initWithOwner:(FlutterTextInputPlugin*)textInputPlugin {
     _markedRect = kInvalidFirstRect;
     _cachedFirstRect = kInvalidFirstRect;
     _scribbleInteractionStatus = FlutterScribbleInteractionStatusNone;
+    _pendingDeltas = [[NSMutableArray alloc] init];
```

Comment:
Do you have to `release` this somewhere?  I'm just guessing since I'm no objective c expert.